src/main/java/com/mateacademy/springmvc/interceptor/UserInterceptor.java

```diff
@@ -0,0 +1,31 @@
+package com.mateacademy.springmvc.interceptor;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@Component
+public class UserInterceptor extends HandlerInterceptorAdapter {
+    private static final Logger LOGGER = LoggerFactory.getLogger("StudentInterceptor");
```

Comment:
1) Maybe it should be changed to  "StudentInterceptor" -> "UsertInterceptor".
2) Why not use Lombok annotation for Logger?